### ISSUE TITLE
Export of segmentations to n5 doesn't use nearest neighbour interpolation

### ISSUE THREAD

**Issue by K-Meech** [mobie/mobie-viewer-fiji#183]
ExportImagePlusAsN5Plugin in Fiji doesn't have options for the interpolation used during downsampling.
For segmentations, this results in values outside the labels for downsampled layers.

Would have to dig into this code https://github.com/bigdataviewer/bigdataviewer-core/blob/master/src/main/java/bdv/export/n5/WriteSequenceToN5.java / or put an issue on their repo.

**Comment by tischi** [comment]
Sounds good, let's do both, modify the code you copied and also raise an issue in the bdv-core repo.

**Comment by tischi** [comment]
@K-Meech 

There are now two PR pending fixing this:

- https://github.com/bigdataviewer/bigdataviewer_fiji/pull/22
- https://github.com/bigdataviewer/bigdataviewer-core/pull/118

If need be, we could think about copying the code to MoBIE until this is merged.

**Comment by K-Meech** [comment]
Great - thanks @tischi!  
Let's leave it for now. I won't have time until the end of next week anyway. So we can always copy it over then, if needs be.

**Comment by K-Meech** [comment]
@tischi Would you be happy to copy this over to MoBIE for now? I should have time next week to make a project creator 2.0 with Matt's suggestions, and it'd be great to include this.

**Comment by tischi** [comment]
@K-Meech Could you first poke Tobias about the PRs? That is, go into the PRs, ping him, and say that you are very interested as well and need this rather soon :)

**Comment by K-Meech** [comment]
@tischi I'm going to start rewriting parts of the project creator soon. Could we copy this over at some point?
I poked Tobias about the PRs, but no response as of yet.

**Comment by tischi** [comment]
I asked in gitter again, let's wait until tomorrow.

**Comment by tischi** [comment]
@K-Meech Tobias will not look into it within the next 2 weeks => copy it over, I guess.

**Comment by K-Meech** [comment]
@tischi I'm copying over this code now, and I noticed an issue.
Currently, if you select a downsampling method it persists throughout that imageJ session i.e. if you export your first image with Average downsampling, then all later images have Average downsampling too.

I added a quick fix here: https://github.com/K-Meech/mobie-viewer-fiji/commit/84a9d1a8f0200a07608b73fd6cc4e6566f959b50
I'm not sure if this affects the h5 export too. Probably worth updating the request on bdv-core?
